src/boot/efi/boot.c

```diff
@@ -396,6 +397,9 @@ static char16_t *update_timeout_efivar(uint32_t *t, bool inc) {
         case TIMEOUT_UNSET:
                 *t = inc ? TIMEOUT_MENU_FORCE : TIMEOUT_UNSET;
                 break;
+        case TIMEOUT_MENU_DISABLED:
+                *t = inc ? TIMEOUT_MENU_FORCE : TIMEOUT_UNSET;
```

Comment:
Hmm, I think we should follow the logic of `menu-hidden` here. Jump from `menu-disabled` directly to `menu-force` seems confusing 🤔